moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -383,7 +398,7 @@ bool JogCalcs::cartesianJogCalcs(geometry_msgs::TwistStamped& cmd, trajectory_ms
   // i.e. take advantage of task redundancy.
   // Remove the Jacobian rows corresponding to True in the vector drift_dimensions
   // Work backwards through the 6-vector so indices don't get out of order
-  for (auto dimension = jacobian.rows(); dimension >= 0; --dimension)
+  for (auto dimension = jacobian.rows() - 1; dimension >= 0; --dimension)
```

Comment:
Hm, shouldn't this have caused more issues?

Reply:
I am not really sure how it did not. The first loop iteration here was trying `drift_dimensions_[dimension]` with the `dimension` out of range, but was returning a `true` instead of throwing an error. I noticed it because I could see all the drift dimensions == 0, but the last row of the Jacobian was being removed anyway